docs/pbc/all/dynamic-multistore/202404.0/base-shop/dynamic-multistore-feature-overview.md

```diff
@@ -80,3 +80,83 @@ Normal storefront APIs can be used to fetch data from stores using the Store HTT
 The following image shows the ability to call API endpoints with the exemplary "SECOND" store selected in the API call itself.
 
 ![storefront-api](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/storefront-api.png)
+
+## Adding stores using data import
+
+
+You can add a new store using data import or in the Back Office. Adding a store in the Back Office is easier and faster, but you have to add this store across all environments.
+
+Using data import, you can configure a new store once and deploy it across all environments.
+
+Example for DE store configuration:
+
+**data/import/common/{REGION}/store.csv**
+
+```csv
+name
+DE
+AT
+```
+
+| COLUMN     | REQUIRED | Data Type | Data Example | Data Explanation |
+|------------| --- | --- | --- | --- |
+|name        | v | string | DE | Define store name. |
+
+Configure a new store as described here: [Import configuration for store](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/install-and-upgrade/install-features/install-dynamic-multistore.html#import-data)
+
+
+When you add a new store, to enable store-related entities for customers, you need to assign them to the store. Some of the store related entities:
+- Products
+- Categories
+- CMS entities
+- Prices
+
+To avoid manually assigning entities in the Back Office, you can assign them using data import. For more details, see
+
+See: [Import minimum set of data for store](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/import-stores.html)
+
+
+## What is changed when Dynamic Multistore is enabled?
+
+Dynamic Multistore enables the following changes to the project:
+
+EU and US regions are used as an example.
+
+EU region has two stores: DE and AT. US region has one store: US.
+
+- URLs for Storefront, Back Office, Merchant Portal and Glue API contain region instead of store name. For example–the URL of the Back Office is `https://backoffice.eu.mysprykershop.com` instead of `https://backoffice.de.mysprykershop.com`.
+
+- RabbitMQ virtual hosts contain region instead of store. For example–`eu-docker` instead of `de-docker`.
+![rabbitmq-virtual-hosts-non-dms](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/rabbitmq-virtual-hosts-non-dms.png)
+![rabbitmq-virtual-hosts-dms](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/rabbitmq-virtual-hosts-dms.png)
+
+- Jenkins job names contain region instead of store. For example–`EU_queue-worker-start` instead of `DE_queue-worker-start`.
+![jenkins-jobs-non-dms](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/jenkins-jobs-non-dms.png)
+![jenkins-jobs-dms](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/jenkins-jobs-dms.png)
```

Comment:
What's the idea with the images?

Reply:
https://docs.spryker.com/docs/about/all/about-the-docs/contribute-to-the-docs/add-images-and-other-media-to-the-docs.html

Reply:
I attached images which should be uploaded . Should I remove image links from docs and leave empty lines instead?